messages/link/errors.go

```diff
@@ -12,7 +12,7 @@ var (
 	ErrorInstallVulcan                 = errors.New("Failed to install Vulcan: %s")
 	ErrorOpeningConfigFile             = errors.New("Failed to open the config.json file. The file doesn't exist, is corrupted, or has an invalid JSON format. Verify if the file format is JSON or fix its content according to the JSON format specification at https://www.json.org/json-en.html")
 	ErrorUnmarshalConfigFile           = errors.New("Failed to unmarshal the config.json file. Verify if the file format is JSON or fix its content according to the JSON format specification at https://www.json.org/json-en.html")
-	ErrorGetAllTags                    = errors.New("Failed returning all Reference tags in a repository. Verify your repository tags and try again. If the error persists, contact Azion support.")
+	ErrorGetAllTags                    = errors.New("Failed returning all reference tags in a repository. Verify your repository tags and try again. If the error persists, contact Azion support.")
```

Comment:
Isn't "Failed to return" better? Then we follow the same pattern of the other messages

Reply:
agreed